ProcessMaker/Providers/ProcessMakerServiceProvider.php

```diff
@@ -164,6 +165,10 @@ public function register(): void
         $this->app->singleton('compiledscreen', function ($app) {
             return new ScreenCompiledManager();
         });
+
+        $this->app->singleton('setting.cache', function ($app) {
```

Comment:
maybe we can validate if cache configuration is enabled ana all necessary settings are present and correct. before to instance to settings cache managger
example

```
public function boot() {
           $this->validateCacheConfig();
       }

       protected function validateCacheConfig() {
           if (!config('cache.default')) {
               throw new \Exception('Default cache configuration is missing.');
           }
       }
```

Reply:
Validation was added